src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -396,7 +418,7 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
   }
 
   private createCardsTabContent(data: IGalleryItem[]): JSX.Element {
-    return (
+    return data ? (
```

Comment:
>data [](start = 11, length = 4)

we are assuming public gallery should not be empty to no show the spinner? why cant we just show the spinner when the api call is made.

Reply:
if public gallery is not empty, users can continue to interact with the notebooks there while we fetch and uodate the cards. So we need to show spinner only on load